14_2_Using_i2p.md

```diff
@@ -0,0 +1,50 @@
+# Chapter 14.2: Using i2p
+
+The Invisible Internet Project (I2P) is a fully encrypted private network layer. It uses a distributed [network database](https://geti2p.net/en/docs/how/network-database) and encrypted unidirectional tunnels between your and your peers.
+
+Basic differences between Tor and i2p: https://geti2p.net/en/comparison/tor
+
+Trade-offs: https://bitcoin.stackexchange.com/questions/107060/tor-and-i2p-tradeoffs-in-bitcoin-core
+
```

Comment:
Ditto for the Trade-offs. 

Just bullet point the biggest few things, and let people click through for the rest.

Reply:
> Just bullet point the biggest few things, and let people click through for the rest.

Sure. Waiting for someone to respond on SE today. If not, will add my own answer and continue as you suggested.

Reply:
Nobody answered on SE. I have added an answer. Also mentioned major trade-offs in bullet points.